.swift-build-linux

```diff
@@ -0,0 +1 @@
+swift build -Xlinker -L/usr/local/lib -Xlinker -lmysqlclient -Xcc -I/usr/include/**
```

Comment:
Why is this file and `.swift-test-linux` not the same?  They have a different include path (`/usr/include/**` vs `/usr/local/include/`)

These files are a Package-Builder thing, not an SPM thing, right?  Is there an SPM mechanism that relieves normal users from having to include the options?